Update drop-in documentation templates (#462)
commerce-docs/microsite-commerce-storefront: _dropin-templates/dropin-installation.mdx

title: DROPIN_NAME installation
description: Learn how to install the DROPIN_NAME drop-in component into your site.
sidebar:
  label: Installation
  order: 2
---

import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
import Tasks from '@components/Tasks.astro';
import Task from '@components/Task.astro';
import Callouts from '@components/Callouts.astro';
import Diagram from '@components/Diagram.astro';
import Vocabulary from '@components/Vocabulary.astro';

> {/* DESCRIBE the DROPIN_NAME drop-in component installation requirements and environment compatibility. Include information about runtime vs build-time environments and bundler support. */}

Reviewer: We don't have use cases for "build-time" environments, nor bundler support in the Commerce Boilerplate. We should base our installation docs on the runtime (browser), which is what we follow in the Commerce Boilerplate.

> {/* DESCRIBE the DROPIN_NAME drop-in component installation requirements and environment compatibility. Include information about runtime vs build-time environments and bundler support. */}

Author: ```suggestion
{/* DESCRIBE the DROPIN_NAME drop-in component installation requirements and environment compatibility. Include information about runtime (browser) environment requirements and any specific configuration needed. /}
```

### Install the packages

> {/* DESCRIBE the package installation requirements for the DROPIN_NAME. Include information about required packages and dependencies. */}

Reviewer: ```suggestion
{/* DESCRIBE the package installation requirements for the DROPIN_NAME. */}
```

There shouldn't be any package dependencies.

<Task>
### Map the packages

> {/* DESCRIBE how to configure the importmap for the DROPIN_NAME. Include information about directory paths and CDN options. */}

Reviewer: ```suggestion
{/* DESCRIBE how to configure the importmap for the DROPIN_NAME. */}
```

We have moved away from deliverying through CDN.
